### ISSUE TITLE
How do we handle the setup / admin fields associated with a specific module?

### ISSUE THREAD

**Issue by tangollama** [HospitalRun/hospitalrun-frontend#175]
@jglovier, we have a design paradigm issue that we need you to weigh in on / decide. We have config / setup data related to a module and only that module, and we need to determine where to put it in the UI. A good example of this is the Billing Module. We have setup data that we need to start capturing like the customizable header and footer information on a bill as well as less-often adjusted data like the Pricing Profiles. One could (and I do) make the argument that putting them in the left nav clutters the navigation with items that will only very rarely be used. Another option is adding a "gear" icon in the module UI to access these items if and only if one has rights to see it.

Bottom line is, we need a decision on how to handle these. Are you game for making this decision?

cc: @jkleinsc

**Comment by jglovier** [comment]
>Bottom line is, we need a decision on how to handle these. Are you game for making this decision?

@tangollama yup, absolutely. I'll dig in when I have some time today or tomorrow. :+1:

**Comment by jglovier** [comment]
After talking with @tangollama last night about this, but still without knowing exactly what type of settings need accommodated for, I've mocked up a generic concept for the Settings UI. Based on what I understand about the requirements, this should work well. However, if I've missed some context or you have details that are important to accommodate for, let me know.

## Default settings screen

![settings-screen](https://cloud.githubusercontent.com/assets/1319791/10142485/f682e492-65e0-11e5-8e8e-37a3c5260fad.png)

## Module subsection

![module-settings-screen](https://cloud.githubusercontent.com/assets/1319791/10142498/06ffd56e-65e1-11e5-9f44-f875edcffe1b.png)

**Comment by jglovier** [comment]
Bump. Anything else you need here? If you spike it out I can style the things, so just hollar when there's some stuff for me to style.
